x-pack/plugins/secops/public/components/notes/note_card/note_created.tsx

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { FormattedRelative } from '@kbn/i18n/react';
+import * as React from 'react';
+import { pure } from 'recompose';
+import styled from 'styled-components';
+
+import { LocalizedDateTooltip } from '../../localized_date_tooltip';
+
+const SelectableText = styled.span`
+  user-select: text;
+`;
```

Comment:
This also exists in `event_details` it looks like. Might be time we pull this out as a comp and just import it for when we need to wrap components in a user-select CSS selector.

Reply:
Good suggestion, `SelectableText` has been refactored into its own component with the following test:

```
  test('it applies the user-select: text style', () => {
    const wrapper = mount(<SelectableText>You may select this text</SelectableText>);

    expect(wrapper).toHaveStyleRule('user-select', 'text');
  });
```